docs/frontend.md

```diff
@@ -3,7 +3,7 @@
 Before doing any development in the front-end, ensure that [Node.js](https://nodejs.org) is installed. To install Node.js, you can do one of the following:
 
 * [Download and install the LTS version](https://nodejs.org/en/download), or
-* [Download and install nvm](https://github.com/creationix/nvm) and follow their instructions to install the Node.js LTS version (recommended), or
+* [Download and install nvm](https://github.com/creationix/nvm) ([win](https://github.com/coreybutler/nvm-windows)) and follow their instructions to install the Node.js LTS version (recommended), or
```

Comment:
I'd propose just saying download nvm and then provide download links for macOs and Windows like this:

```
* Download and install nvm ([macOS](https://github.com/creationix/nvm), [Windows](https://github.com/coreybutler/nvm-windows)) and follow the installation instructions
```

Reply:
Good catch on the missing Windows download link though. 👍 I missed that in the initial version of these docs.

Reply:
I have no strong opinion but this is a great start. Thanks for bring nvm-windows into our radar, definitely will add to 
our [WIP windows doc](https://docs.dev.to/installation/windows/) and it will cover nvm-windows there.